LinkifiedText/index.jsx

```diff
@@ -49,6 +49,7 @@ LinkifiedText.propTypes = {
       indices: PropTypes.arrayOf(React.PropTypes.number),
     }),
   ),
+  size: PropTypes.string,
```

Comment:
Curious if we might want to have this set to something like `size: PropTypes.oneOf(['extra-small', 'large', 'small']),` to ensure that we can't pass any other string in here. 🤔 Not too sure if this check is made higher up the chain i.e. `'small'` gets passed in to `LinkifiedText`, which then gets passed through to `Text` and then the check is made there? It feels like that _might_ be the case but not 100% sure! 😅